security/remote_devices/docker-swarm/puttoken.sh

```diff
@@ -0,0 +1 @@
+docker exec -u root -ti `docker ps  --filter name=device-virtual | awk '{ print $1 }' | tail -n 1` sh -c 'mkdir -p /tmp/edgex/secrets/device-virtual ; chmod 777 /tmp/edgex/secrets/device-virtual ; cat > /tmp/edgex/secrets/device-virtual/secrets-token.json'
```

Comment:
How is this getting the vault token??

Reply:
gettoken.sh gets the token.  puttoken.sh writes the token (from stdin, pasted from the user) to the second node.  Referenced in the documentation.